lib/tasks/export.rake

```diff
@@ -0,0 +1,7 @@
+namespace :export do
+  desc 'Generate CSV file with Single Offender IDs paired with NOMIS ids'
+  task offenders: :environment do
+    file = Rails.root.join('lib', 'assets', 'data', 'soi_extract.csv')
```

Comment:
**[not a blocker]** 💭 interface wise, it could be better to have the service to deal its conventions internally (e.g. file location, file name) and only allow it to be passed as an option if strictly necessary:

```ruby
# by default file location would be lib/assets/data and file name a time version soi_extract_20170222124525.csv
ExportOffenders.call

# with options - this might not be required at all at this point in time
ExportOffenders.call(base_filename: 'my_own_name', location: '/tmp/)
```